enhancements/arbiter-clusters.md

```diff
@@ -0,0 +1,597 @@
+---
+title: arbiter-clusters
+authors:
+  - "@eggfoobar"
+reviewers:
+  - "@tjungblu"
+  - "@patrickdillon"
+  - "@racedo"
+  - "@deads2k"
+  - "@jerpeter1"
+  - "@sjenning"
+  - "@yuqi-zhang"
+  - "@zaneb"
+  - "@rphillips"
+approvers:
+  - "@tjungblu"
+  - "@patrickdillon"
+  - "@racedo"
+  - "@jerpeter1"
+  - "@deads2k"
+  - "@sjenning"
+  - "@yuqi-zhang"
+  - "@zaneb"
+api-approvers:
+  - "@JoelSpeed"
+creation-date: 2024-08-27
+last-updated: 2024-10-16
+tracking-link:
+  - https://issues.redhat.com/browse/OCPEDGE-1191
+see-also: []
+replaces: []
+superseded-by: []
+---
+
+# Support 2 Node + 1 Arbiter Node HA Cluster
+
+## Summary
+
+This enhancement describes an ability to install OpenShift with a control plane
+that consists of at minimum 2 normal sized nodes, and at least 1 node that can
+be less powerful than the recommended node size. This 1 arbiter node will only
+be running critical components for maintaining HA to allow the arbiter node size
+to be as small and as low cost as possible with in reason.
+
+## Motivation
+
+Customers at the edge are requiring a more economical solution for HA
+deployments at the edge. They can support running 2 node clusters for redundancy
+but would like the option to deploy a lower cost node as an arbiter to supply
```

Comment:
It would also be worth considering having an arm64 node for the arbiter to lower costs and deploy the cluster with the multi-arch payload.

Reply:
is the arbiter node considered strictly control plane?

Reply:
AIUI we know the specific hardware we are targeting here (it has the low-powered node built-in).